src/gigs-board/components/posts/Post.jsx

```diff
@@ -486,10 +655,26 @@ const postTitle =
 const postExtra =
   snapshot.post_type == "Sponsorship" ? (
     <div key="post-extra">
-      <h6 class="card-subtitle mb-2 text-muted">
+      <h6
+        class="card-subtitle  text-muted"
+        style={{
+          marginLeft: "1rem",
+          marginBottom: "1rem",
+          fontSize: "1.25789em",
```

Comment:
Why this fractional size?

Reply:
sorry I do that sometimes when working to make it easier to find them quickly, changing it back now.